packages/@aws-cdk/aws-ec2-alpha/lib/transit-gateway-vpc-attachment.ts

```diff
@@ -0,0 +1,126 @@
+import { ITransitGateway, TransitGatewayFeatureStatus } from './transit-gateway';
+import { CfnTransitGatewayAttachment, ISubnet, IVpc } from 'aws-cdk-lib/aws-ec2';
+import { Construct } from 'constructs';
+import { TransitGatewayRouteTableAssociation } from './transit-gateway-route-table-association';
+import { TransitGatewayRouteTablePropagation } from './transit-gateway-route-table-propagation';
+import { ITransitGatewayAttachment, TransitGatewayAttachmentBase } from './transit-gateway-attachment';
+
+export interface ITransitGatewayVpcAttachment extends ITransitGatewayAttachment {}
+
+export interface ITransitGatewayVpcAttachmentOptions {
+  /**
+   * Enable or disable appliance mode support.
+   *
+   * @default - disable (false)
+   */
+  readonly applianceModeSupport?: boolean;
+
+  /**
+   * Enable or disable DNS support.
+   *
+   * @default - disable (false)
+   */
+  readonly dnsSupport?: boolean;
+
+  /**
+   * Enable or disable IPv6 support.
+   *
+   * @default - disable (false)
+   */
+  readonly ipv6Support?: boolean;
+
+  /**
+   * Enables you to reference a security group across VPCs attached to a transit gateway.
+   *
+   * @default - disable (false)
+   */
+  readonly securityGroupReferencingSupport?: boolean;
+}
+
+export interface TransitGatewayVpcAttachmentProps {
+  /**
+   * A list of one or more subnets to place the attachment in.
+   * It is recommended to specify more subnets for better availability.
+   */
+  readonly subnets: ISubnet[];
+
+  /**
+   * The transit gateway this attachment gets assigned to.
+   */
+  readonly transitGateway: ITransitGateway;
+
+  /**
+   * A VPC attachment(s) will get assigned to.
+   */
+  readonly vpc: IVpc;
+
+  /**
+   * The VPC attachment options.
+   */
+  readonly transitGatewayVpcAttachmentOptions?: ITransitGatewayVpcAttachmentOptions;
```

Comment:
```suggestion
  readonly vpcAttachmentOptions?: ITransitGatewayVpcAttachmentOptions;
```

This is in transitGateway file already, I think we can omit the prefix.

Reply:
You're right, this should be fine to omit the prefix.